docs/content/version-reporting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Anonymous Version Reporting
+kind: operations
+weight: 50
+---
+
+This document provides details about OPA's anonymous version reporting feature.
+
+## Overview
+
+OPA supports anonymous version reporting wherein OPA would reach out to an external service and share its version 
+information. This feature is only applicable to the `opa run` and `opa version` commands.
+
+In case of the `opa run` command, this feature will be **ON by-default** and can be easily disabled by specifying
+the `--incognito` flag. When OPA is started in either the `server` or `repl` mode, OPA would call the external service
```

Comment:
Nit: don't need backticks for server and repl--they're not subcommands.

Reply:
Done